### PR TITLE
[ng] toh-1: fix class name; show what happens when directives are missing

### PR DESCRIPTION
**Pull request by chalin**
- Fixed error in prose: `Hero` class name was used instead of `AppComponent`
- Show reader what error s/he will get when ng encounters a directive that it doesn't recognize (the template compiler throws an error).
- Other misc formatting updates to `toh-1` code.

@kwalrath, as usual, you can ignore the patch file

### CODE REVIEW

File: src/angular/tutorial/toh-pt1.jade [hunk 1]
```diff
@@ -129,6 +129,20 @@ include keep-app-running
 :marked
   ### @Component(directives: ...)
 
+  Unfortunately, immediately after this change, the **application breaks**!
+  If you open the browser console and refresh the page, you'll see Angular
+  complaining.  To know why, look at the `pub serve` output. The template
+  comipler doesn't recognize `ngModel`, and issues a parse error for
```
Comment (kwalrath) [inline src/angular/tutorial/toh-pt1.jade]: comipler -> compiler

File: src/angular/tutorial/toh-pt1.jade [hunk 2]
```diff
@@ -129,6 +129,20 @@ include keep-app-running
...
+  `AppComponent` because it
```
Comment (kwalrath) [inline src/angular/tutorial/toh-pt1.jade]: Let's complete this sentence.

because it -> :

File: src/angular/tutorial/toh-pt1.jade (hunk 1 above)
Comment (chalin) [inline src/angular/tutorial/toh-pt1.jade]: fixed

File: src/angular/tutorial/toh-pt1.jade (hunk 2 above)
Comment (chalin) [inline src/angular/tutorial/toh-pt1.jade]: fixed

**Pull request merged**